integration/apps/build_func.sh

```diff
@@ -90,3 +95,35 @@ unpack_archive() {
         tar xvf ${ARCHIVE}
     fi
 }
+
+# Clear out previously installed binaries
+clean_geopm() {
+    if [ -e "${GEOPM_INSTALL}" ]; then
+        echo "WARNING: Previous install of geopm or other data found at ${GEOPM_INSTALL}"
+        read -p "OK to delete and reinstall? (y/n) " -n 1 -r
+        echo
+        if [[ ${REPLY} =~ ^[Yy]$ ]]; then
+            rm -rf ${GEOPM_INSTALL}
+        else
+            echo "Not OK.  Stopping."
+            return 1
+        fi
+    fi
+}
+
+# Build geopm in a subdirectory and install
+install_geopm() {
+    local BASE_DIR=${PWD}
+    local BUILD_DIR=${GEOPM_SOURCE}/integration/build
```

Comment:
BUILD_DIR should be checked to see if there's an old build there already, and if yes, blow it away.